lib/Models/LocationMarkerUtils.js

```diff
@@ -44,7 +45,7 @@ export function addMarker(terria, viewState, result) {
         correctEntityHeight(firstPointEntity, cartographicPosition, terria.cesium.scene.globe.terrainProvider, 15);
```

Comment:
Not something you changed, but it'd be great to use `prettifyCoordinates` to generate the entity's description, too.  Currently it uses `${result.location.latitude}, ${result.location.longitude}`.

Reply:
Have a look at  88ed3b3538ece4d174f8d0f2ea302407e015e658 and let me know if this is what you were thinking. I've preserved 5 dp since it seems nice to preserve the additional resolution in the description (testing revealed that about 5 dp was detailed enough precision compared with the visual base map without providing more precision then is apparent).
![image](https://user-images.githubusercontent.com/4453477/34861118-9d4d0b90-f7b6-11e7-8226-4001e6cba988.png)